rosetta/services/transaction_parser.go

```diff
@@ -52,10 +58,24 @@ func (tp *transactionsParser) parseTx(eTx *data.FullTransaction, isInPool bool)
 }
 
 func (tp *transactionsParser) createRosettaTxFromUnsignedTx(eTx *data.FullTransaction) (*types.Transaction, bool) {
+	// TODO check if we have a SCR that call another contract
 	if eTx.Value == "0" {
 		return nil, false
 	}
 
+	switch {
+	case eTx.GasLimit != 0:
+		// we have a SCR with gas refund
+		return tp.createRosettaTxWithGasRefund(eTx)
+	case eTx.Sender != eTx.Receiver:
```

Comment:
Well, one SC can also send funds to itself.

Anyway, we can live with this for the moment.

Added a task: 

https://elrondnetwork.atlassian.net/browse/EN-7896